docs/installation/server-config/security.rst

```diff
@@ -4,20 +4,100 @@ Securing KSQL
+        consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+        consumer.confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+        consumer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        consumer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        consumer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        consumer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        consumer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+    The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+    are signed. For full details, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+#.  Start KSQL server with your configuration file specified.
+
+    .. code:: bash
+
+        $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+    .. tip:: After KSQL is started, you can view your settings with the ``SHOW PROPERTIES;`` KSQL command.
+
+---------------------------------------------------
+Using KSQL with a secured Confluent Schema Registry
+---------------------------------------------------
+
+If you are using KSQL with HTTPS to Confluent Schema Registry, you must set the ``ksql.schema.registry.url`` to HTTPS
+(as shown in the example configuration) and set the ``KSQL_OPTS`` environment variable to define the credentials to use
+when communicating with the Confluent Schema Registry:
+
+.. code:: bash
+
+    # Define KSQL security credentials when communicating with the Confluent Schema Registry via HTTPS
+    $ export KSQL_OPTS="-Djavax.net.ssl.trustStore=/etc/kafka/secrets/kafka.client.truststore.jks
+                        -Djavax.net.ssl.trustStorePassword=confluent
+                        -Djavax.net.ssl.keyStore=/etc/kafka/secrets/kafka.client.keystore.jks
+                        -Djavax.net.ssl.keyStorePassword=confluent"
+
 -------------------------------------------------
-Using KSQL with an Apache Kafka Secured with ACLs
+Using KSQL with a Kafka Cluster Secured with ACLs
 -------------------------------------------------
 
 You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
 interactive or non-interactive.
 
+.. tip:: For more information about ACLs see :ref:`kafka_authorization` and for more information about interactive and
+         non-interactive queries, see :ref:`streams_developer-guide_interactive-queries`.
```

Comment:
Sorry, I meant KSQL's deployment configuration (interactive/default vs. non-interactive/headless configuration), not KStreams' interactive queries.

There's actually, in this file, a section called "Interactive KSQL clusters" just down below...

Reply:
updated link to `restrict-ksql-interactive` (e.g. https://github.com/confluentinc/ksql/blob/8cfd40a4de62222923c45a436f50c15c4a8d7582/docs/installation/server-config/index.rst).